src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Refactor/IRefactorNotifyService.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Refactor
+{
+    internal interface IRefactorNotifyService
+    {
+        Task<bool> TryOnBeforeGlobalSymbolRenamedAsync(string projectPath, IEnumerable<string> filePaths, string rqName, string newName);
+        Task<bool> TryOnAfterGlobalSymbolRenamedAsync(string projectPath, IEnumerable<string> filePaths, string rqName, string newName);
```

Comment:
These return values are always ignored.

Reply:
Yep I thought we might want to handle the failure case when I was writing this but I realized later it didn't matter